samples/snippets/requirements.txt

```diff
@@ -1 +1,2 @@
-google-cloud-secret-manager==2.9.0
\ No newline at end of file
+crcmod>=1.7
```

Comment:
There is also a possibility to use `google-crc32==1.3.0` from https://github.com/googleapis/python-crc32c instead of `crcmod`. WDYT?

```
    # Calculate payload checksum. Passing a checksum in add-version request
    # is optional.
    crc32c = google_crc32c.Checksum()
    crc32c.update(payload)

    # Add the secret version.
    response = client.add_secret_version(
        request={
            "parent": parent,
            "payload": {"data": payload, "data_crc32c": int(crc32c.hexdigest(), 16)}
        }
    )
```